app/models/session.js

```diff
@@ -0,0 +1,7 @@
+import DS from 'ember-data';
+import Session from 'exp-models/models/session';
+
+export default Session.extend({
+    frameIndex: DS.attr({defaultValue: 0}),
```

Comment:
As discussed previously, the model should provide clear documentation for the meaning of special fields, especially in the case where the meaning is unclear. A source code comment here would be a valuable addition.